src/hexen/h2_main.c

```diff
@@ -676,6 +676,23 @@ static void HandleArgs(void)
         ST_Message("Playing demo %s.\n", myargv[p+1]);
     }
 
+    //!
+    // @category demo
+    //
+    // Record or playback a demo with high resolution turning.
```

Comment:
Heretic/Hexen demos are equivalent to Doom's 1.2 demos: there's no header information that includes the game version.

Heretic+ seems to be limit-removing only; Hexen+ has improved demo support and longtics, but playing back a demo recorded with `-longtics` desyncs immediately.

However, I just took a look at some demos created through vvHeretic and they appear to use the header byte that checks for the existence of player one to store extra information. 0x20 indicates `-respawn`, 0x10 indicates `-longtics`, and 0x02 indicates `-nomonsters` (nothing for `-fast`, probably because it's very similar to the Black Plague difficulty). I'm almost positive we can assume player one will exist in every demo, so I'll try to imitate the usage in recording and playback.